politeiawww/cmd/politeiawwwcli/commands/censorcomment.go

```diff
@@ -8,6 +8,28 @@ import (
 	"github.com/decred/politeia/util"
 )
 
+// Help message displayed for the command 'politeiawwwcli help censorcomment'
+var CensorCommentCmdHelpMsg = `censorcomment "token" "commentID" "reason"
+
+Censor comment (admin).
```

Comment:
I would write out what `(admin)` means.  Something like:

`Censor a user comment.  Requires admin privileges.`

Same for all commands that require admin privileges.

Reply:
Put `Requires admin privilages` on all commands that require admin. The only one I wasn't sure of was `edituser`, where I broke it out by bit flag (copied current copy in comment below).